client/src/components/SentimentAnalysis/Helpers.js

```diff
@@ -0,0 +1,81 @@
+import { COLOR_MAP, INPUT_ROWS } from '../../constants/analysis_constants';
+
+import PropTypes from 'prop-types';
+import React from 'react';
+
+const AnalysisInput = props => {
+  const { label, placeholder } = props;
+
+  return (
+    <div className="search-header center">
+      <h3 className="analysis-label">{label}</h3>
+      <textarea
+        type="text"
+        rows={INPUT_ROWS}
+        className="ad-input"
+        placeholder={placeholder}
+      />
+    </div>
+  );
+};
+
+AnalysisInput.propTypes = {
+  label: PropTypes.string,
+  placeholder: PropTypes.string,
+};
+
+const ColorBar = props => {
+  const { score, magnitude } = props;
+
+  const colorObject = COLOR_MAP.neutral;
+
+  if (score < 0.0) {
+    Object.assign(colorObject, COLOR_MAP.negative);
+  } else if (score > 0.0) {
+    Object.assign(colorObject, COLOR_MAP.positive);
+  }
+
+  return (
+    <div
+      className="color-bar"
+      style={{
+        color: colorObject.color,
+        backgroundColor: colorObject.background,
+      }}
+    >
+      {`${colorObject.label}: ${score} ----- Magnitude: ${magnitude}`}
+    </div>
+  );
+};
+
+ColorBar.propTypes = {
+  score: PropTypes.number,
+  magnitude: PropTypes.number,
+};
+
+const TermsDisplay = props => {
+  const { termList } = props;
+
+  return (
+    <div className="analysis-text-container">
+      {termList.map((termString, index) => {
+        const termObject = JSON.parse(termString);
+        return (
+          <div key={index} className="center">
+            <div className="analysis-text" id="special-text">
```

Comment:
It seems strange to me to have a helper that has an ID in it.  IDs should be unique, meaning this helper would only ever be used once and you might as well just inline it.